### DISCUSSION TITLE
Cannot find path for "assets/bootstrap.css" asset. Make sure you are compiling assets

### DISCUSSION THREAD

**Discussion by vikrampunchh** [adonisjs/core#3847] [Help]
I am using Assets manager and setup this.

I have added  resources/css/bootstrap.css file but getting an error

terminal output

```[ encore ]  DONE  Compiled successfully in 661ms12:15:48 PM
[ encore ] webpack compiled successfully
UPDATE: public/assets/manifest.json
UPDATE: public/assets/entrypoints.json
```

Error:
Cannot find path for "assets/bootstrap.css" asset. Make sure you are compiling assets

**Comment by Barbapapazes** [comment]
Hello,

Can you provide more information on how you're loading this file or a reproduction repository?

Thanks!

**Reply by vikrampunchh** [reply]
I am using it in development 

```  
<link rel="stylesheet" type="text/css" href="{{ asset('assets/bootstrap.css') }}" />
```

` node ace serve --watch`

**Reply by Barbapapazes** [reply]
Can you tell me whère you store the bootstrap.css file? 

`public/assets/...`
or
`ressources/...`

**Reply by Barbapapazes** [reply]
For a css file, you can use an entry point in a js file https://docs.adonisjs.com/guides/assets-manager#entrypoints
